DataManagementSystem/Agent/DataLoggingAgent.py

```diff
@@ -0,0 +1,26 @@
+'''
+Created on Jun 19, 2015
+
+@author: Corentin Berger
+'''
+from DIRAC import S_OK
+
+from DIRAC.Core.Base.AgentModule import AgentModule
+from DIRAC.DataManagementSystem.Client.test.mockDirac import ClientB
+
+# # agent's name
+AGENT_NAME = 'DataManagement/DataLoggingAgent'
+
+########################################################################
+class DataLoggingAgent( AgentModule ):
+  def initialize( self ):
```

Comment:
Sorry, I forgot to remove this file, it was to test something, it is not needed anymore.